### PR TITLE
Implement card functionality when linking to product story on twitter

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit aa9b27bfa9c24d2e09108a01369ea302a1b54fe8

https://deploy-preview-537--unboxed-co-production.netlify.com

File: source/product-stories/chargy.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Product Story - char.gy - Providing residents with an on-street electric vehicle lamppost charging solution with char.gy"
+title: "Product Story - char.gy"
```
Comment (h-lame) [inline source/product-stories/chargy.html.erb:2]: Do we want the removed bit as the `meta_description`?  Looks like that's what we've done on other product stories.

**Review by h-lame — APPROVED** [review]
Other than a minor thing about one `meta_description`, this looks good.  Checked it on https://cards-dev.twitter.com/validator and we now have cards for pages we didn't have before.  Yay!

File: source/product-stories/chargy.html.erb (hunk 1 above)
Comment (cdccollins) [reply source/product-stories/chargy.html.erb:2]: Yep good point, I'll add it in

**Pull request merged**